### ISSUE TITLE
NPM package out of sync

### ISSUE THREAD

**Issue by gerwing** [yannickcr/node-synology#2]
When getting this package via NPM the code seems to be out of sync with the code on github. In the package.json, api.js is set as the main file and nothing else than app.js is included in the lib folder.

**Comment by yannickcr** [comment]
You are right, I's been a while since I did not update the module. I will soon take back the development and update the npm package.

**Comment by redimongo** [comment]
is this still in production, or should we move on, would be happy to help with the documentation part if needed.
